contracts/src/Tracker.sol

```diff
@@ -10,6 +10,7 @@ struct Challenge {
     address verifier;
     uint256 minimumCheckIns;
     uint256 startTimestamp;
+    uint256 joinDueTimestamp;
     uint256 endTimestamp;
```

Comment:
make all these uint64, reasons
* packing into a single slot for gas efficiency
* `viem` fetch it as `number` instead of `bigint`, no additional casting on the fe